OpenRA.Game/Game.cs

```diff
@@ -457,6 +465,28 @@ static string ChooseShellmap()
 			return shellmaps.Random(CosmeticRandom);
 		}
 
+		public static void SwitchToExternalMod(ExternalMod mod, string[] launchArguments = null, Action onFailed = null)
+		{
+			try
+			{
+				var argsString = mod.LaunchArgs.Append(launchArguments)
+					.Select(a => "\"" + a + "\"").JoinWith(" ");
+
+				var p = Process.Start(mod.LaunchPath, argsString);
```

Comment:
You are still re-inventing software distribution without making sure that
* the software comes from a trustworthy source (code signing)
* if source code is available: proof that the binary comes from that source (reproducible builds)

which makes this dangerous. No matter if for now the mods are installed manually or in the future become one-click available from an online repository.

For OpenRA maps we did a huge fuss about security by design with sandboxed Lua and no other runtimes or libraries allowed. For OpenRA mods you are going this dangerous route. :skull_and_crossbones: